### DISCUSSION TITLE
Poetry not using the default python version

### DISCUSSION THREAD

**Discussion by michaelxswe** [python-poetry/poetry#9548] [Q&A]
As you can see from the image, my current python version is 3.11,9, but it says The currently activated Python version 3.12.4 is not supported by the project (3.11). I'm not sure how to set poetry to use the right version

![Screenshot 2024-07-14 183921](https://github.com/user-attachments/assets/1ee95d0d-7d2c-4ccb-aa46-7093b8f065ad)

Is there a way to have poetry be in sync with the current python version?

**Comment by sbrunner** [comment]
Poetry is probably installed in Python 3.12, running `pip install poetry` (or `python -m pip install poetry`) will probably solve your issue.

**Comment by dimbleby** [comment]
please read the docs https://python-poetry.org/docs/configuration#virtualenvsprefer-active-python-experimental
